src/backend/joanie/tests/core/api/organizations/test_api_organizations_contract.py

```diff
@@ -155,6 +155,7 @@ def test_api_organizations_contracts_list_with_accesses(self, _):
                     },
                     "order": {
                         "id": str(contract.order.id),
+                        "state": contract.order.state,
```

Comment:
You can probably add a test where you completely control the value of the state and not relying on `contract.order.state` ?